src/views/Correction.vue

```diff
@@ -143,11 +145,11 @@ export default class Correction extends Vue {
   /** Emits Fetch Error event. */
   @Emit('fetchError')
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  private emitFetchError (err: unknown = null): void {}
+  emitFetchError (err: unknown = null): void {}
 
   /** Emits Have Data event. */
   @Emit('haveData')
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  private emitHaveData (haveData = true): void {}
```

Comment:
I got rid of these because my TS/Vue extensions complain that the template can't see these methods because they're private or protected. But it doesn't do this with all components and I don't know why...